Cargo.lock

```diff
@@ -249,6 +249,7 @@ dependencies = [
 name = "hieratika-flo"
 version = "0.1.0"
 dependencies = [
+ "anyhow",
```

Comment:
Should this be local or a workspace dep? Seems like we may want to harmonize it across modules.